js/ascendex.js

```diff
@@ -2149,7 +2183,7 @@ module.exports = class ascendex extends Exchange {
         this.checkAddress (address);
```

Comment:
Hi, Thank you for following up with this issue. 
Just wondering if parseDepositAddress should also be updated too as the API response is as below for this part?

```
       //                {
        //                    "address": "0xaa9f2c949f56ad1cf3cee7c314da8d06b44c8c4a",
        //                    "destTag": "",
        //                    "blockchain": "ERC20"
        //                },

```
tag should be = destTag?